internal/builder/v1beta1/pod.go

```diff
@@ -124,9 +123,6 @@ func PodContainer(name, image string, ops ...ContainerOp) PodSpecOp {
 		c := &corev1.Container{
 			Name:  name,
 			Image: image,
-			Resources: corev1.ResourceRequirements{
-				Requests: map[corev1.ResourceName]resource.Quantity{},
-			},
```

Comment:
I am not sure why this was needed to be changed to fix the issue with some tests but... meh.